src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -43,7 +43,7 @@ const Breadcrumb = ({ crumbs, currentPage, texts, hasOverflow = true }: Breadcru
         {overflowCrumbs.length > 0 && (
           <li className="bsds-breadcrumb-overflow">
```

Comment:
thoughts on where to draw the line for this change? i can see the use case for a singular crumb when we're takling about one item vs the group of breadcrumbs. what was your thought process for leaving breadcrumbNavAriaLabel and bsds-breadcrumb-overflow as singular?